index.js

```diff
@@ -61,56 +62,17 @@ const argv = yargs(process.argv.slice(2))
         type: 'boolean',
         default: false
     })
+    .option('concurrency', {
+        describe: 'Number of parallel downloads',
+        type: 'number',
+        default: 10

```

Comment:
this feels a tad too high, we risk triggering some sort of ddos protection, i'd lover it to 4